.github/workflows/go-test.yml

```diff
@@ -13,8 +13,8 @@ jobs:
     name: go-test
     strategy:
       matrix:
-        go-version: [1.21.x, 1.22.x]
-        platform: [ubuntu-latest, macos-latest]
```

Comment:
We probably want to leave `macos-latest` here, since we're explicitly targeting it, and have specific tests around it

Reply:
Restored